robot/impl/local_robot.go

```diff
@@ -619,7 +619,12 @@ func New(
 	return newWithResources(ctx, cfg, nil, logger, opts...)
 }
 
-func (r *localRobot) newService(ctx context.Context, config config.Service) (interface{}, error) {
+func (r *localRobot) newService(ctx context.Context, config config.Service) (res interface{}, err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			err = errors.Errorf("panic creating resource: %v", r)
```

Comment:
I'd use errors.Wrap on both here. also service not resource

Reply:
panics don't usually return errors so I'll make a new error